src/test/resources/application.properties

```diff
@@ -0,0 +1,7 @@
+spring.datasource.url=jdbc:h2:mem:testdb
+spring.datasource.driverClassName=org.h2.Driver
+spring.datasource.username=sa
+spring.datasource.password=password
+spring.jpa.database-platform=org.hibernate.dialect.H2Dialect
+spring.h2.console.enabled=true
+spring.jpa.show-sql=true
```

Comment:
you shouldn't have changed test resources. By default if there is no application.properties within test folder the one from main folder is used